package.json

```diff
@@ -39,13 +39,14 @@
     "test:ember-compatibility": "ember try:each"
   },
   "dependencies": {
+    "@glimmer/tracking": "^1.0.4",
+    "broccoli-replace": "^2.0.2",
```

Comment:
I think this can be removed

Reply:
Oops, totally forgot about that. Removed it!